plugin-server/tests/main/ingestion-queues/session-recording/fixtures.ts

```diff
@@ -1,21 +1,24 @@
 import { IncomingRecordingMessage } from '../../../../src/main/ingestion-queues/session-recording/types'
+import { RRWebEvent } from '../../../../src/types'
 import jsonFullSnapshot from './data/snapshot-full.json'
 
 export function createIncomingRecordingMessage(
     partialIncomingMessage: Partial<IncomingRecordingMessage> = {},
-    partialMetadata: Partial<IncomingRecordingMessage['metadata']> = {}
+    partialMetadata: Partial<IncomingRecordingMessage['metadata']> = {},
+    partialFirstSnapshotEvent: Partial<RRWebEvent> = {}
 ): IncomingRecordingMessage {
     // the data on the kafka message is a compressed string.
     // it is a compressed $snapshot PostHog event
     // that has properties, and they have $snapshot_data
     // that will have data_items, which are the actual snapshots each individually compressed
 
+    const snapshotEvent: RRWebEvent = { ...jsonFullSnapshot, ...partialFirstSnapshotEvent }
```

Comment:
What is the effect of this change?

Reply:
Here `jsonFullSnapshot` is a fixed representation of an `RRWebWvent` event. In the past this was fine, but now we have the replay ingestion in this code path too we care about the timestamp because that ingestion path checks that the timestamp isn't too far in the past.

This lets me override the snapshot timestamp while keeping the args to this method simiar